lib/Config/LocationAwareConfigRepository.php

```diff
@@ -129,6 +143,21 @@ public function getWriteTarget(): string
         return $writeLocation;
     }
 
+    public function getReadTargets(): array
```

Comment:
@robertSt7 what is this used for? Seems to be unused?

Reply:
@brusch Initial thought was to have something for the future, since we plan on having more than one read target for different bundles. And to somehow access the read targets from outside the class, if necessary. 
But of course we could/should use it inside the class as well, e.g. https://github.com/pimcore/pimcore/pull/14889/files#diff-aea6f63bf38b8863f731103a59f9a169a7ebd704dd223cb40a3813b1f94b737bR60

Reply:
see https://github.com/pimcore/pimcore/pull/14928